torch/csrc/jit/passes/peephole.cpp

```diff
@@ -52,6 +52,18 @@ struct PeepholeOptimizeImpl {
         run(sub_block);
       }
 
+      if (node->kind() != prim::Constant) {
+        WithInsertPoint guard(node);
+        // Any Value whose type is None should be replaced with a Constant
+        // This can occur if a module has an optional attribute, and it is
+        // initialized as None.
+        for (Value* output : node->outputs()) {
```

Comment:
As far as I can think of, this will pretty much only occur with a module attribute. However I guess a mutating custom op could return None as well